doc/api/vm.md

```diff
@@ -511,11 +511,11 @@ in the ECMAScript specification.
   * `timeout` {integer} Specifies the number of milliseconds to evaluate
     before terminating execution. If execution is interrupted, an [`Error`][]
     will be thrown. This value must be a strictly positive integer.
-  * `breakOnSigint` {boolean} If `true`, the execution will be terminated when
-    `SIGINT` (Ctrl+C) is received. Existing handlers for the event that have
-    been attached via `process.on('SIGINT')` will be disabled during script
-    execution, but will continue to work after that. If execution is
-    interrupted, an [`Error`][] will be thrown. **Default:** `false`.
+  * `breakOnSigint` {boolean} If `true`, execution is terminated when `SIGINT`
+    (<kbd>Ctrl</kbd>+<kbd>C</kbd>) is received. Existing handlers for the
+    event that have been attached via `process.on('SIGINT')` are disabled
+    during script execution, but continue to work after that. If execution
+    is interrupted, an [`Error`][] is thrown. **Default:** `false`.
```

Comment:
The word `interrupted` here is the one thing I didn't modify for consistency. The other six instances say `terminated`. I wasn't completely sure the choice wasn't intentional or which word was better than the other. @nodejs/vm @nodejs/documentation

Reply:
I think interrupt is a good word, especially given that that’s what `SIGINT` stands for :)

Reply:
Should it be `interrupted` on the other six instances then? I think consistency would be good, and @addaleax  makes a good point.

Reply:
> Should it be `interrupted` on the other six instances then?

I'll land this as-is and open a follow-on PR about that.